src/api/utils/summary_cache.py

```diff
@@ -282,6 +284,29 @@ def _update_unrestricted_project_summary_sample_events(
     )
 
 
+def add_project_to_queue(project_id, skip_test_project=False):
+    try:
+        if isinstance(project_id, uuid.UUID) is False:
```

Comment:
use `check_uuid()`?

Reply:
I was on the fence if I should use this.  I always associate it with returning a 400 error response but I agree it's likely worth using check_uuid